.github/CONTRIBUTING.md

```diff
@@ -75,6 +75,114 @@ Also, have you signed the [Contribution License Agreement](https://cla.microsoft
 
 **If your contribution is going to be a major effort, you should give us a heads-up first. We have a lot of items captured in our backlog and we release every two weeks, so before you spend the time, just check with us to make sure your plans and ours are in sync :) Just open an issue on GitHub and tag it as "contribution".**
 
+#### Commit Guidelines
```

Comment:
Question - Why do we have 2 different md files for contributing guidelines? - `./CONTRIBUTING.md` and `./.github/CONTRIBUTING.md`
We should remove the one that is not required.

Reply:
Good observation :) Let me see which one we want to keep.